docs/UserGuide.md

```diff
@@ -124,6 +136,12 @@ Adds a student to the TutorBuddy
 Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEVEL g/GUARDIAN_PHONE_NUMBER r/RELATIONSHIP_WITH_GUARDIAN` <br>
 * `STUDENT_PHONE_NUMBER`, `GUARDIAN_PHONE_NUMBER` should be in Singapore's phone formatting (i.e. starting with either 6, 8 or 9 and 8 digits)
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+TutorBuddy only allows students with distinct names, however, its case sensitive. 
+Hence, students with the same name with different cases are treated as non-duplicated students. <br>
+(E.g. TutorBuddy allows students with both John Doe and john doe)
```

Comment:
This is not related to your UG, but practicality wise, isn't it funny to have us accept same name but different cases? HAHA

Reply:
Haha yeah ah. But I think if we want to change, we need to update it at alot of places though. So I put it here as a "feature", so people can't dispute with us, saying its a bug ah.

Reply:
Anw if you want, we could create an issue, then if we have time then we can work on that ah. If not then we will just go along with this?